build.gradle

```diff
@@ -81,6 +81,7 @@ dependencies {
 
   compile project(':email-client')
   compile project(':domain-model')
+  compile project(':ccd-client')
```

Comment:
I can't see anything cmc specific in this module so far so I believe @geek-soft 's comment from the previous PR still stands:
https://github.com/hmcts/cmc-claim-store/pull/114#discussion_r154402803

Reply:
Other thing is that ccd-client should be generic and should have no knowledge of mapping. Here `ccd-client` depends on `ccd-adapter'` which makes it unusable for anyone else.

Reply:
https://github.com/hmcts/core-case-data-store-client/releases/tag/1.0.0

Reply:
Great, now we have to use it ;)